test/onnx/test_fx_to_onnx.py

```diff
@@ -330,6 +330,7 @@ def forward(self, input):
                 expected_node="aten.mul.Tensor",
             )
 
+    @pytorch_test_common.xfail(error_message="'submodule_1' not found in {")
```

Comment:
@BowenBao is the value info propagated by the optimize or something like that?

Reply:
What likely happened is the simple local function is optimized away and broke this test assumption.